docs/src/components/careers/CareersFaq.tsx

```diff
@@ -69,62 +69,57 @@ const faqData = [
   {
     summary: 'Does MUI do whiteboarding during interviews?',
     detail:
-      'No. We ask applicants to complete challenges that are close to their future day-to-day contributions.',
+      'No. We ask applicants to complete challenges that resemble the kinds of contributions we would expect from the role.',
   },
   {
-    summary: 'Contractor or Employee?',
+    summary: 'Would I be hired as an employee or contractor?',
```

Comment:
To have them sync with https://www.notion.so/mui-org/Hiring-FAQ-64763b756ae44c37b47b081f98915501 (same data)